k8s/netmanager-app/templates/deployment.yaml

```
@@ -0,0 +1,55 @@
+  minReadySeconds: 5
+  selector:
+    matchLabels:
+      app: {{ .Values.app.label }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.app.label }}
+    spec:
+      containers:
+        - name: {{ .Values.app.label }}
+          securityContext: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)